bin/lib/create.js

```diff
@@ -58,8 +58,7 @@ module.exports.createProject = function (project_path, package_name, project_nam
     shell.cp('-r', path.join(ROOT, 'bin/template/www'), project_path);
 
     // recreate our node_modules structure in the new project
-    shell.cp('-r', path.join(ROOT, 'node_modules'),
-        path.join(project_path, 'cordova'));
+    if (options && options.copyPlatformNodeModules) shell.cp('-r', path.join(ROOT, 'node_modules'), path.join(project_path, 'cordova'));
```

Comment:
minor: I would have split it up:

```suggestion
    if (options && options.copyPlatformNodeModules)
        shell.cp('-r', path.join(ROOT, 'node_modules'), path.join(project_path, 'cordova'));
```